editor/editor_settings.cpp

```diff
@@ -1017,10 +1018,30 @@ void EditorSettings::save() {
 	if (err != OK) {
 		ERR_PRINT("Error saving editor settings to " + singleton->config_file_path);
 	} else {
+		singleton->changed_settings.clear();
 		print_verbose("EditorSettings: Save OK!");
 	}
 }
 
+const Array &EditorSettings::get_changed_settings() const {
+	return changed_settings;
+}
```

Comment:
Does this method have any use? (aside from convenient debugging 🤔)

Reply:
For binding to the class db?

Reply:
It was a long time, but I probably meant that this method isn't used anywhere. However `check_changed_settings_in_group()` theoretically doesn't cover all cases, because you might want to check specific setting, so maybe it's ok to have it, idk